README.md

```diff
@@ -79,6 +81,9 @@ This is achieved via profile activation (**pi**, "push image"), eg:
 ```
 mvn clean install -pl <module name> -P pi
 ```
+This will only create new charts and Docker images with a new tag value.
+These new charts and images will still need to be deployed to any current helm deployments of Palisade.
+This can be done by using the `helm upgrade` command or by doing a new `helm install` if Palisade has not been deployed.
 
 ### Considerations for running under Windows
 
```

Comment:
Missing the text or remove the header

Reply:
The section below this header contains the considerations for running under Windows.